app/src/main/java/org/sopt/and/data/PreferencesManager.kt

```diff
@@ -2,8 +2,9 @@ package org.sopt.and.data
 
 import android.content.Context
 import android.content.SharedPreferences
+import javax.inject.Inject
 
-class PreferencesManager(context: Context) {
+class PreferencesManager @Inject constructor(context: Context) {
```

Comment:
아래와 같이 바꾸고 NetworkModule 을 한 번 지워볼까요!?
```suggestion
class PreferencesManager @Inject constructor(@ApplicationContext context: Context) {
```

Reply:
@ApplicationContext를 사용하면 Hilt가 자동으로 애플리케이션의 Context를 주입해주기 때문에, PreferencesManager와 같은 클래스에서 별도의 모듈을 통해 제공할 필요가 없습니다. 라고 나오네요. 감사합니다! 
좋은 정보 감사합니다. 이 부분은 따로 내용 정리해서 공부하면 좋을것 같습니다.